domain_sslinfo.py

```diff
@@ -13,8 +13,11 @@ def check_ssl_htbsecurity(domain):
 	headers['Content-Type'] = "application/x-www-form-urlencoded"
 	data='domain=%s&dnsr=off&recheck=false' % domain
 	req = requests.post('https://www.htbridge.com/ssl/chssl/1451425590.html', headers=headers , data=data)
-	results = json.loads(req.content)
-	return results
+	if req.content:
+		results = json.loads(req.content)
+		return results
+	else:
+		return "Empty JSON"
```

Comment:
Returning a `string` "Empty JSON" instead of a list doesn't make sense. Then in the `main()` function, there will be an error when `results.keys()` is executed.

@ckennard I think you have to **implement** an **error handling** technique to stay on the safer side.